.travis.yml

```diff
@@ -17,6 +17,9 @@ cache:
   - $HOME/build_tools
   - $HOME/.cache/pip
 
+before_install:
+ - "sudo apt-get install graphviz"
```

Comment:
Is this required? Can we just add graphviz to the requirements.txt file?

Reply:
But "requirements.txt" is only for Python pip, right?  This is the actual Graphviz for the Linux machine.

Reply:
O I see. It requires the installation for rendering and the python package doesn't include the full install.